sail/src/main/java/org/apache/rya/rdftriplestore/RdfCloudTripleStore.java

```diff
@@ -2,12 +2,6 @@
 
 import static com.google.common.base.Preconditions.checkNotNull;
 
-import org.openrdf.model.ValueFactory;
-import org.openrdf.model.impl.ValueFactoryImpl;
-import org.openrdf.sail.SailConnection;
-import org.openrdf.sail.SailException;
-import org.openrdf.sail.helpers.SailBase;
-
 /*
  * Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
nit.  move the header above the package line as was done in other files.

Reply:
Done.